static/js/custom_script.js

```diff
@@ -1,3 +1,38 @@
+
+
 $(document).ready(function(){
     $("#id_cmd_aliases").select2();
-  });
\ No newline at end of file
+});
+
+
+function cloneMore(selector, type) {
+    
+    //var newElement = $().clone(true);
+    var Element = $("#" + selector).children("ul").last();
+    if (Element.size() <= 0 ) {
+        return; // No element to clone.
+    }
+    
+    var newElement = Element.clone(true);
+    
+    var total = $('#id_' + type + '-TOTAL_FORMS').val();
+    total++;
+    
+    newElement.find(':input').each(function() {
```

Comment:
Here the element ID increment need to be fixed. Right now the new formsets are being added on click. Also need to test duplicate addition of the forms since we are managing the form additions via javascript.

Need to test the POST request once the form creation is done.